src/Teams/Models/ApplicationDbContext.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Teams.Models
+{
+    public partial class ApplicationDbContext : DbContext
```

Comment:
Why we using partial class? We already have ApplicationDbContext with all DbSets (Users, Claims, Roles etc.). This is totally useless class.